libs/ha/ha_common_libs_k8s.py

```diff
@@ -1907,3 +1929,45 @@ def form_endpoint_port(pod_obj, pod_list, port_name="rgw-https"):
             pod_ep_dict[pod] = ip_port
 
         return pod_ep_dict
+
+    def create_iam_user_with_header(self, i_d, header):
+        """
+        Function create IAM user with give header info.
+        :param i_d: Int count number for IAM user name creation
+        :param header: Existing header to use for IAM user creation post request
+        :return: None if IAM user REST req fails or Dict response for IAM user successful creation
+        """
+        user = None
+        payload = {}
```

Comment:
make this dict()

Reply:
> make this dict()

This is giving me warning
Consider using {} instead of dict()  (use-dict-literal)